src/OpenTK.Windowing.Desktop/NativeWindow.cs

```diff
@@ -816,19 +818,19 @@ private void RegisterWindowCallbacks()
                         GLFW.GetJoystickButtonsRaw(joy, out var buttonCount);
                         var name = GLFW.GetJoystickName(joy);
 
-                        JoystickStates[joy] = new JoystickState(hatCount, axisCount, buttonCount, joy, name);
+                        _joystickStates[joy] = new JoystickState(hatCount, axisCount, buttonCount, joy, name);
```

Comment:
These subsequent usages of _joystickStates instead of JoystickStates are for consistency.